Enterprise-CMCS/managed-care-review
# MR-3232: Resubmitting does not update the status tag on the dashboard immediately

## Summary
[MR-3232](https://qmacbis.atlassian.net/browse/MR-3232)

Updating the Apollo cache at submit and resubmit so that when redirected, the status of the submission is up-to-date while waiting for the fresh data to come back from the API.

#### Related issues

#### Screenshots

#### Test cases covered



## QA guidance

- Testing this locally, you can force an error in the emailer functions.
- Testing this on the review app, you can:
   - Complete a submission up till review and submit.
   - Use graphQL Explorer and get the submission revision formDataProto.
   - Decode the proto, edit the state contact email to an invalid email format (i.e. emailWithNoDomain), and encode.
   - Use the Explorer tool and use `updateHealthPlanFormData` to update.
   - Now when you go back to the review and submit page the email will be in an invalid format.
   - Clicking, submit you should be redirected to the dashboard page, and in the console or network tab you should have seen an error.
